extensions/notebook/src/test/python/jupyterInstallation.test.ts

```diff
@@ -193,4 +201,92 @@ describe('Jupyter Server Installation', function () {
 		let commandStr = commandStub.args[0][0] as string;
 		should(commandStr.includes('"jupyter==1.0.0" "TestPkg2==4.5.6"')).be.true();
 	});
+
+	it('Get required packages test', async function() {
+		// Undefined argument
+		let packages = installation.getRequiredPackagesForKernel(undefined);
+		should(packages).not.be.undefined();
```

Comment:
Please either add description messages to all of these checks so when/if it fails it's clear what one failed or split these up into separate tests